tracer/src/Datadog.Trace/AppSec/Waf/NullOkResult.cs

```diff
@@ -0,0 +1,39 @@
+// <copyright file="NullOkResult.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace Datadog.Trace.AppSec.Waf
+{
+    internal class NullOkResult : IResult
+    {
+        public NullOkResult()
+        {
```

Comment:
Is it worth creating a singleton of this, if it's never going to change, so you can re-use across requests?
```suggestion
    internal class NullOkResult : IResult
    {
        public static readonly Instance = new NullOkResult();

        private NullOkResult()
        {
```

Reply:
deleted in subsequent commits